lib/providers/services/firebase/firebase_database_service.dart

```diff
@@ -295,8 +295,7 @@ class FirebaseDatabaseService {
     return batch.commit();
```

Comment:
this file is huge, and all service calls are here. 

Lets move this into smaller repos like we do with network calls.